### ISSUE TITLE
javac path problem

### ISSUE THREAD

**Issue by APOORVMAGAN** [mwrlabs/drozer#198]
Could not find java. Please ensure that it is installed and on your PATH.

If this error persists, specify the path in the ~/.drozer_config file:

```
[executables]
java = C:\path\to\java
```

There was a problem connecting to the drozer Server
...............................................................................................................
I have done following changes in Lib\drozer\configuration.py but it does not work
print name
    #check the library
if name == 'javac':  
    path = 'C:\Program Files\Java\jdk1.7.0_75\bin\javac.exe'
else:  
    path = cls.library(name)

**Comment by HenryHoggard** [comment]
Please create the file .drozer_config in your home directory and set the Java path.
You have set your javac path rather than the java path.
```
[executables]
java = C:\Program Files\Java\jdk1.7.0_75bin\java.exe
```

**Comment by HenryHoggard** [comment]
As this appears to be a common issue, we will work on automatically finding the path for Windows users in future versions.

**Comment by APOORVMAGAN** [comment]
Hi,
still same problem is there

**Comment by APOORVMAGAN** [comment]
![drozer](https://cloud.githubusercontent.com/assets/14202709/13944007/802c721e-f027-11e5-8fbb-07df7f411790.png)
![drozer2](https://cloud.githubusercontent.com/assets/14202709/13944016/8bb30058-f027-11e5-8178-b53de4636f37.png)
![drozer3](https://cloud.githubusercontent.com/assets/14202709/13944036/b98a0c56-f027-11e5-9ecc-97072bee8103.png)

**Comment by albertKharkov** [comment]
try to put both java and javac 

`[executables]
java = C:\Program Files\Java\jdk1.6.0_45\bin
javac = C:\Program Files\Java\jdk1.6.0_45\bin\javac.exe`

because you are talking about problem with javac but put PATH to java instead javac

**Comment by amarekano** [comment]
Hi all, this issue has now been resolved with PR #270. This was due to a bug in the way drozer located executables on Windows. Our next drozer release (version 2.4.3) will include this fix. 

If you urgently need to get drozer working, please clone our develop branch and build drozer from source.

Thank you once again for reporting this issue and contributing to improving drozer. :)
